### PR TITLE
File upload done

### PR DESCRIPTION
**Pull request by muizece**
File upload done using multer 
Launch.json added for debugging and 
options.js added for jwt token

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -21,13 +25,22 @@ const userSchema = new mongoose.Schema({
     email: String,
     password: String,
     id: Number,
+    role:String,
```
Comment (ismailbaig) [inline index.js]: what is this role, is it any new column in mongodb. If yes, send that file as well.

File: index.js [hunk 2]
```diff
@@ -21,13 +25,22 @@ const userSchema = new mongoose.Schema({
...
 
 const Users = new mongoose.model("User", userSchema);
 
 // Mongoose Schema - End
 
+
+// View Engine Setup
+// app.set("views",path.join(__dirname,"views"))
+
+app.set("views","views")
```
Comment (ismailbaig) [inline index.js]: should there be a "app.set("views",path.join(__dirname,"views"))"

File: index.js [hunk 3]
```diff
@@ -40,7 +53,8 @@ app.post("/login", (req, res) => {
         if(user) {
             let userdata = {
                 username: uid,
-                password: pd
+                password: pd,
+                role:user.role
```
Comment (ismailbaig) [inline index.js]: what is this role ?

File: index.js [hunk 4]
```diff
@@ -153,6 +233,8 @@ function verifyToken(req, res, next) {
   
 }
 
+
```
Comment (ismailbaig) [inline index.js:221]: clean up.

**Review by ismailbaig — COMMENTED** [review]
please look into review comments

File: index.js (hunk 1 above)
Comment (muizece) [reply index.js]: I will  send user.json file ,upload this in your mongodb

File: index.js (hunk 2 above)
Comment (muizece) [reply index.js]: Yes, I will do it

File: index.js (hunk 3 above)
Comment (muizece) [reply index.js]: I will send user.json file ,upload this in your mongodb

File: index.js (hunk 4 above)
Comment (muizece) [reply index.js:221]: Yes , i will do the changes

**Pull request merged**